.goreleaser.yml

```diff
@@ -28,47 +28,67 @@ changelog:
     exclude:
       - '^docs:'
       - '^test:'
+release:
+  prerelease: auto
```

Comment:
Need to delete the `release` key section at the top of this file since it's duplicated here.